plugins/lsp/default.nix

```
@@ -16,6 +16,11 @@ in
     plugins.lsp = {
       enable = mkEnableOption "neovim's built-in LSP";
 
+      package = lib.mkPackageOption pkgs [
+        "vimPlugins"
+        "nvim-lspconfig"
+      ];
```

```suggestion
      ] { };
```

This will error as "partially applied function assigned to option" or however its worded